editor/layout/index.js

```diff
@@ -33,8 +33,8 @@ function Layout( { mode, isSidebarOpened, notices, ...props } ) {
 		'is-sidebar-opened': isSidebarOpened,
 	} );
 
-	return (
-		<div className={ className }>
+	return [
```

Comment:
Why is this switched to an array?

Reply:
I must have missed that while rebasing. The Metabox used to be added after the editor area in an array, now it is inside. Good catch.